src/model/bpmn/shape/ShapeBpmnElementKind.ts

```diff
@@ -42,6 +42,8 @@ export enum ShapeBpmnElementKind {
   // TODO: Uncomment corresponding test in test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.conditional.test.ts
   // TASK_BUSINESS_RULE = 'businessRuleTask',
```

Comment:
```suggestion
  // TODO: Uncomment corresponding test in test/unit/component/parser/json/BpmnJsonParser.label.bounds.test.ts
  // TODO: Uncomment corresponding test in test/unit/component/parser/json/BpmnJsonParser.label.font.test.ts
  // TODO: Uncomment corresponding test in test/unit/component/parser/json/BpmnJsonParser.label.test.ts
  // TASK_BUSINESS_RULE = 'businessRuleTask',
```

Do the same for **TASK_SEND**, **TASK_SCRIPT**,  **TASK_MANUAL** & **GATEWAY_COMPLEX**